spec.md

```diff
@@ -1339,6 +1339,21 @@ message Error {
       // information to detach the volume from one other node before
       // retrying with exponential backoff.
       MAX_ATTACHED_NODES = 8;
+
+      // Indicates that the specified `NodeID` is not allowed or
+      // understood by the Plugin. More human-readable information MAY
+      // be provided in the `error_description` field.
+      //
+      // Recovery behavior: Caller MUST fix the `NodeID` before
+      // retrying.
+      INVALID_NODE_ID = 9;
+
+      // Indicates that the `NodeID` is not set to match that returned
+      // by `GetNodeID`.
+      //
+      // Recovery behavior: Caller MUST fix the `NodeID` before
+      // retrying.
+      NODE_ID_REQUIRED = 10;
```

Comment:
Hmm... make sense. I'll remove it and explain in the comments of `INVALID_NODE_ID` that it includes the case that the `NodeID` is missing.